components/Login/LoginButton.tsx

```diff
@@ -0,0 +1,18 @@
+import styles from "./LoginButton.module.css";
+
+function LoginButton() {
+  function handleClick(e) {
+    e.preventDefault();
+    console.log("You will be redirected to spotify.");
+  }
+
+  return (
+    <button className={`${styles.login}`}>
+      <img src="/spotify-icon.svg" className={`${styles.spotify}`} />
+      <a href="#" onClick={handleClick}>
```

Comment:
Why use an `a` tag?

Reply:
There will be a link to spotify authentication later.